### PR TITLE
kk_mock_populate: Create users and comments

### PR DESCRIPTION
**Pull request by akx**

### CODE REVIEW

File: kk/factories/hearing.py [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
 import random
+from django.contrib.auth import get_user_model
 from django.utils.timezone import now
+from kk.factories.comment import BaseCommentFactoryMixin
 from kk.models import Hearing, Label, Scenario
+from kk.models.hearing import HearingComment
+from kk.models.scenario import ScenarioComment
+
+
+def get_random_user():
```
Comment (rphlo) [inline kk/factories/hearing.py]: defined also in factory utils

File: kk/factories/hearing.py [hunk 2]
```diff
@@ -39,6 +58,10 @@ def post(obj, create, extracted, **kwargs):
             scenario = ScenarioFactory(hearing=obj)
             print(".. Created scenario %s" % scenario.pk)
 
+        for x in range(random.randint(1, 5)):
+            comment = HearingCommentFactory(hearing=obj)
```
Comment (rphlo) [inline kk/factories/hearing.py]: Missing update of nb_comment field

File: kk/factories/hearing.py (hunk 1 above)
Comment (akx) [inline kk/factories/hearing.py]: Copypaste fail. Will fix.

File: kk/factories/hearing.py (hunk 2 above)
Comment (akx) [inline kk/factories/hearing.py]: Good catch, adding

**Pull request merged**